files/image_config/resolv-config/update-containers

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 
-for container in $(docker ps -a --format=" {{ .ID }}"); do
+networking_status=$(systemctl is-active networking.service 2>/dev/null)
+if [[ $networking_status != "active" ]]; then
+    exit 0
+fi
+
+for container in $(docker ps -q); do
```

Comment:
Why do you use "docker ps -q" to replace "docker ps -a"?

Reply:
`docker ps -a` returns a list of all available containers (active and inactive). `docker ps -q` returns a list of running containers. We want to update only running containers. For the inactive containers docker will uptade `/etc/resolv.conf` file automatically during the container start. So they will always have up-to-date configuration.